src/Panel/Panel.php

```diff
@@ -54,45 +53,51 @@ public static function area(string $id, array $area): array
     public static function areas(): array
     {
         $kirby  = kirby();
-        $root   = $kirby->root('kirby') . '/config/areas';
         $system = $kirby->system();
         $user   = $kirby->user();
+        $areas  = $kirby->load()->areas();
 
         // the system is not ready
         if ($system->isOk() === false || $system->isInstalled() === false) {
             return [
-                'installation' => static::area('installation', (require $root . '/installation.php')($kirby)),
+                'installation' => static::area('installation', $areas['installation']),
             ];
         }
 
         // not yet authenticated
         if (!$user) {
             return [
-                'login' => static::area('login', (require $root . '/login.php')($kirby)),
+                'login' => static::area('login', $areas['login']),
             ];
         }
 
-        // load default areas
-        $areas = [
-            'site'     => static::area('site', (require $root . '/site.php')($kirby)),
-            'settings' => static::area('settings', (require $root . '/settings.php')($kirby)),
-            'users'    => static::area('users', (require $root . '/users.php')($kirby)),
-            'account'  => static::area('account', (require $root . '/account.php')($kirby)),
-        ];
+        unset($areas['installation'], $areas['login']);
+
+
+        $menu = $kirby->option('panel.menu', [
+            'site',
+            'settings',
+            'users'
+        ]);
+
+        $result = [];
 
-        // load plugins
-        foreach ($kirby->extensions('areas') as $id => $area) {
-            if (is_a($area, 'Closure') === false) {
-                throw new InvalidArgumentException(sprintf('Panel area "%s" must be defined as a Closure', $id));
```

Comment:
We don't need this check anymore?

Reply:
After thinking about it, there's actually no harm in defining an area or an extension of a part of an area as a simple array when you don't need access to the $kirby var.

Reply:
So far there was a harm because a simple array wouldn't work and cause problems. If now a simple array is supported, indeed we should allow that.

Reply:
Tested and works now. 👍